mountpoint-s3-fs/src/superblock/inode.rs

```diff
@@ -344,9 +344,9 @@ impl InodeStat {
     /// restored, and so we override their permissions to 000 and reject reads to them. We also warn
     /// the first time we see an object like this, because FUSE enforces the 000 permissions on our
     /// behalf so we might not see an attempted `open` call.
-    fn is_readable(storage_class: Option<String>, restore_status: Option<RestoreStatus>) -> bool {
+    fn is_readable(storage_class: Option<&str>, restore_status: Option<RestoreStatus>) -> bool {
```

Comment:
`storage_class` is still stored as `String` right? (don't see it changed to `Box<str>`)

Reply:
We do not store it in the inodes, it's only used here to determine the `is_readable` flag.